Scripts/Runtime/Data/Collections/UnsafeTypedStream.cs

```diff
@@ -526,6 +611,14 @@ public struct LaneReader
             private BlockInfo* m_CurrentReadBlock;
             private byte* m_EndOfCurrentBlock;
             private byte* m_ReaderHead;
+            
+            /// <summary>
+            /// Reports if this <see cref="LaneReader"/> has been created and is valid to use.
+            /// </summary>
+            public bool IsCreated
+            {
+                get => m_BufferInfo != null;
```

Comment:
💿 same comment about adding an assert to the constructor.

Reply:
Ensured that constructor can't be called without passing through the assert.